tools/reduce.py

```diff
@@ -268,31 +284,27 @@ def main():
     # TODO: add --hang option to detect code which impacts the analysis time
     def show_syntax():
         print('Syntax:')
-        print('  reduce.py --cmd=<full command> --expected=<expected text output> --file=<source file> [--segfault]')
+        print('  reduce.py --exe <cppcheck executable> --exe_args <full command> --expected <expected text output> --file <source file> [--segfault]')
         print('')
         print("Example. source file = foo/bar.c")
         print(
-            "  reduce.py --cmd='./cppcheck --enable=style foo/bar.c' --expected=\"Variable 'x' is reassigned\" --file=foo/bar.c")
+            "  reduce.py --exe \"./cppcheck\" --exe_args \"\\-\\-enable=style\" --expected \"Variable 'x' is reassigned\" --file \"foo/bar.c\"")
```

Comment:
I suggest:

    '  reduce.py --exe ./cppcheck --exe_args " --enable=style" --expected "Variable \'x\' is reassigned" --file foo/bar.c')